runtime/oti/FFIUpcallThunkGenHelpers.hpp

```diff
@@ -0,0 +1,303 @@
+#if !defined(FFIUPCALLTHUNKGENHELPERS_HPP_)
```

Comment:
This file needs a license and copyright.

Reply:
Should be fixed now.